x/evm/types/utils.go

```diff
@@ -46,8 +46,9 @@ func ValidateSigner(signBytes, sig []byte, signer ethcmn.Address) error {
 func rlpHash(x interface{}) (hash ethcmn.Hash) {
 	hasher := sha3.NewLegacyKeccak256()
 
-	// nolint:errcheck
-	rlp.Encode(hasher, x)
+	if err := rlp.Encode(hasher, x); err != nil {
+		_ = fmt.Errorf("error with RLP encoding %e", err)
```

Comment:
Is there no way to add a comment to avoid linting this instead of doing all of this non-logic just to pass the rule?

Reply:
yeah we can do that